pkg/entities/user_context.go

```diff
@@ -88,6 +88,15 @@ func (u UserContext) GetIntAttribute(attrName string) (int64, error) {
 	return 0, fmt.Errorf(`no int attribute named "%s"`, attrName)
 }
 
+// GetAttribute returns the value for the specified attribute name in the attributes map. Returns error if not found.
+func (u UserContext) GetAttribute(attrName string) (interface{}, error) {
```

Comment:
Why added this method?

Reply:
for `InvalidAttributeValueType` log, we required the raw attribute value to log its original type.